### PR TITLE
docs : Added note about DB performance to LDAP

### PR DESCRIPTION
**Pull request by yash2189**
#### Summary

* This PR adds a note section to the [LDAP Sync Interval configuration docs](https://docs.mattermost.com/administration/config-settings.html#synchronization-interval-minutes) so that customers can know regarding the database performance while LDAP Synchronization happens.
* Changes have been made under `source/administration/config-settings.rst`
* Please find the updated file [here](https://github.com/yash2189/docs/blob/MM-3630/source/administration/config-settings.rst)

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2600,6 +2600,9 @@ When accounts are disabled in AD/LDAP users are made inactive in Mattermost, and
 | This feature's ``config.json`` setting is ``"SyncIntervalMinutes": 60`` with numerical input.                                                                        |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+.. note::
+  Because LDAP syncs cause a large number of database read queries you should monitor database load during a sync to determine how often these syncs should happen in your environment.
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:2604]:
```suggestion
  LDAP syncs cause a large number of database read queries. Ensure that you monitor database load during a sync to determine how often these syncs should happen in your environment in order to minimize performance degradation.
```

**Comment by yash2189** [comment]
@justinegeffen The changes have been made.

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks, @yash2189! :)

**Comment by justinegeffen** [comment]
Once the PM review is completed we can merge the PR. :)

**Review by thefactremains — APPROVED** [review]

**Pull request merged**